source/postprocess/dynamic_topography.cc

```diff
@@ -257,7 +257,15 @@ namespace aspect
                              "in the outputted data file (not visualization). "
                              "'true' subtracts the mean, 'false' leaves "
                              "the calculated dynamic topography as is. ");
-
+          prm.declare_entry ("Density above","0",
+                             Patterns::Double (0),
+                             "Dynamic topography is calculated as the excess or lack of mass that is supported by mantle flow. "
+                             "This value depends on the density of material that is moved up or down, i.e. crustal rock, and the "
+                             "density of the material that is displaced (generally water or air). While the density of crustal rock "
+                             "is part of the material model, this parameter 'density_above' allows the user to specify the density "
```

Comment:
Please replace the name of the variable by the name of the parameter, i.e. change 'density_above' to 'Density above' in the description.